examples/todoapp/.gitignore

```diff
@@ -33,6 +33,7 @@ lib/assets.dart
 **/*.gr.dart
 **/*.rxb.g.dart
 **/*.mocks.dart
+**/*.realm.dart
```

Comment:
In the official MongoDB Realm [documentation](https://www.mongodb.com/docs/atlas/device-sdks/sdk/flutter/realm-database/model-data/define-realm-object-schema/#generate-realmobject) it is recommended to track the `.realm.dart` files using git.

Reply:
Done